tests/safety/test_toyota.py

```diff
@@ -117,6 +124,59 @@ def test_accel_actuation_limits(self, stock_longitudinal=False):
             should_tx = np.isclose(accel, 0, atol=0.0001)
           self.assertEqual(should_tx, self._tx(self._accel_msg(accel)))
 
+  def test_steer_req_bit_frames(self):
+    """
+      On Toyota, we set the STEER_REQUEST bit to 0 every 19 messages while above 100 deg/s
+      to avoid a steering fault and maintain torque. This tests:
+        - We can't cut torque for more than one frame
+        - We can't cut torque until at least 19 frames of matching steer_req messages
+        - We can always recover from violations if steer_req=1
+    """
+    for steer_rate_frames in range(MIN_VALID_STEERING_FRAMES * 2):
+      # reset rt timer and match count to allow cut
+      self._reset_toyota_timer()
```

Comment:
probably better to use `self.safety.init_tests()`

Reply:
think self.setUp should work, init_tests doesn't reset anything we want

Reply:
you don't really want to call setUp manually. you can just add these things to init_tests